src/components/shell/shell.tsx

```diff
@@ -11,8 +11,10 @@ import {
  * @slot - A slot for adding content to the shell. This content will appear between any leading and trailing panels added to the shell. (eg. a map)
  * @slot header - A slot for adding header content. This content will be positioned at the top of the shell.
  * @slot footer - A slot for adding footer content. This content will be positioned at the bottom of the shell.
- * @slot primary-panel - A slot for adding the leading `calcite-shell-panel`.
- * @slot contextual-panel - A slot for adding the trailing `calcite-shell-panel`.
+ * @slot panel-start - A slot for adding the starting `calcite-shell-panel`.
+ * @slot panel-end - A slot for adding the ending `calcite-shell-panel`.
+ * @deprecated @slot primary-panel - A slot for adding the leading `calcite-shell-panel`.
```

Comment:
Is this the correct way to deprecate a slot? cc @benelan

Reply:
cc @macandcheese @geospatialem you both found out the jsdoc tag doesn't work for slots as well as css vars right? IIRC we use `[Deprecated]` and then parse it out and add the chip in the doc.

> @ slot primary-panel - [Deprecated] A slot for adding the leading `calcite-shell-panel`.

(added a space between `@` and `slot` so I didn't tag some random guy)

Reply:
@driskull I confirmed that adding `[DEPRECATED]` to the description is the correct way to deprecate slots and css vars. We will get that added to the documentation conventions.